src/main/java/net/moonbyte/AnarchyUtils/Commands/ChangeNameColorCommand.java

```diff
@@ -39,79 +40,94 @@ public boolean onCommand(@NotNull CommandSender commandSender, @NotNull Command
             // here goes the monkey code
             if(color.equalsIgnoreCase("black")){
                 nameColor = ChatColor.BLACK.getChar();
-                PlayerUtils.sendMessage(p, ConfigurationUtils.getConfig().getString("nc-command.success-message").replaceAll("<color>", "black"));
+                assert ConfigUtil.NoPermissionMessageDonation != null;
```

Comment:
I believe this enire area is incorrect.

First we are checking if ConfigUtil.NoPermissionMessageDonation is not null in each if statement. The null check for this should be within ConfigUtils, or outside of the if statement so we don't have to duplicate code.

Another thing is that we don't show a error message or anything if it is null. We should.

Last, we are only showing the NoPermissionMessageDonation, we're not showing the success message. This seems like it was not tested at all.